### PR TITLE
Fix windows_task configuration

### PR DESCRIPTION
**Pull request by smurawski**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -36,4 +36,4 @@
 
 depends 'cron', '>= 1.2.0'
 depends 'logrotate', '>= 1.2.0'
-depends 'windows', '~> 1.11'
+depends 'windows', '~> 1.37'
```
Comment (juliandunn) [inline metadata.rb]: What's the motivation for this pin?

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Manages client.rb configuration and chef-client service'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.3.0'
+version           '4.4.0'
```
Comment (juliandunn) [inline metadata.rb]: Can you avoid updating version numbers in metadata? I'll do that when ready to release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+v4.4.0 (2015-06-25)
```
Comment (juliandunn) [inline CHANGELOG.md]: Just leave this as `UNRELEASED` rather than pinning to a version number plz

**Comment by alexpop** [comment]
![(shipit)]
(http://qph.is.quoracdn.net/main-qimg-0bf1d3de309a89d9ed70ae91f6271a7f)

File: metadata.rb (hunk 1 above)
Comment (jaym) [inline metadata.rb]: There were many bug fixes to tasks in that release

**Comment by alexpop** [comment]
@juliandunn, I made the changes requested. Hoping to merge this in for the next release.
It fixes the windows_task recipe.

**Comment by juliandunn** [comment]
LGTM.

**Pull request closed without merging**